### PR TITLE
{bio}[foss/2015a] BayPass 2.1

### PR DESCRIPTION
**Pull request by AFCJamie**
An easyconfig file for BayPass 2.1 using the foss 2015a toolchain.

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2607.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/3d9c32aa4de1edb5a887d4d9eecfde26 for a full test report.

File: easybuild/easyconfigs/b/BayPass/BayPass-2.1-foss-2015a.eb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+easyblock = 'MakeCp'
+
+name = 'BayPass'
+version = '2.1'
```
Comment (boegel) [inline easybuild/easyconfigs/b/BayPass/BayPass-2.1-foss-2015a.eb]: @AFCJamie also here: trailing whitespace, please remove?

File: easybuild/easyconfigs/b/BayPass/BayPass-2.1-foss-2015a.eb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+homepage = 'http://www1.montpellier.inra.fr/CBGP/software/baypass/'
+description = """The package BayPass is a population genomics software which is primarily aimed at identifying genetic markers subjected to selection and/or associated to population-specific covariates (e.g., environmental variables, quantitative or categorical phenotypic characteristics)."""
```
Comment (boegel) [inline easybuild/easyconfigs/b/BayPass/BayPass-2.1-foss-2015a.eb]: @AFCJamie we try to limit line length to 120 characters or less, please break up this long line

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
gligar01.gligar.os - Linux SL 6.8, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/423f487ae53bdd088dd673a520d86d30 for a full test report.

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2499.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/11f77d88671b14696fd79a056b6532b2 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @AFCJamie!

**Comment by AFCJamie** [comment]
Happy days, thanks for your patience with me!

**Comment by boegel** [comment]
Thank *you* for contributing back, keep 'em coming! :)

**Pull request merged**